test/tests/ver-1kc-2/comparison_ver-1kc-2.py

```diff
@@ -0,0 +1,184 @@
+equilibrium_constant_encl_1 = expt_data_k10['equilibrium_constant_encl_1']
+equilibrium_constant_encl_2 = expt_data_k10['equilibrium_constant_encl_2']
+
+
+# Subplot 1 : Pressure vs time
+
+fig = plt.figure(figsize=[6.5,5.5])
+gs = gridspec.GridSpec(1,1)
+ax = fig.add_subplot(gs[0])
+
+# Plot for Enclosure 1
+line1 = ax.plot(TMAP8_time_k10, TMAP8_pressure_H2_enclosure_1_k10, label=r"H$_2$ Enclosure 1", c='tab:red', linestyle='-')
+line2 = ax.plot(TMAP8_time_k10, TMAP8_pressure_T2_enclosure_1_k10, label=r"T$_2$ Enclosure 1", c='tab:orange', linestyle='--')
+line3 = ax.plot(TMAP8_time_k10, TMAP8_pressure_HT_enclosure_1_k10, label=r"HT Enclosure 1", c='tab:purple', linestyle='-')
+
+ax.yaxis.set_major_formatter(plt.FuncFormatter(lambda val, pos: '{:.1e}'.format(val)))
+ax.set_xlabel('Time (s)')
+ax.set_ylabel('Pressure Enclosure 1 (Pa)')
+ax.set_xlim(0, TMAP8_time_k10.max())
+ax.set_ylim(bottom=0)
+ax.grid(which='major', color='0.65', linestyle='--', alpha=0.3)
+
+# Plot for Enclosure 2
+ax2 = ax.twinx()
+line4 = ax2.plot(TMAP8_time_k10, TMAP8_pressure_H2_enclosure_2_k10, label=r"H$_2$ Enclosure 2", c='tab:blue', linestyle='-')
+line5 = ax2.plot(TMAP8_time_k10, TMAP8_pressure_T2_enclosure_2_k10, label=r"T$_2$ Enclosure 2", c='tab:green', linestyle='--')
+line6 = ax2.plot(TMAP8_time_k10, TMAP8_pressure_HT_enclosure_2_k10, label=r"HT Enclosure 2", c='tab:cyan', linestyle='-')
+
+ax2.yaxis.set_major_formatter(plt.FuncFormatter(lambda val, pos: '{:.1e}'.format(val)))
+ax2.set_ylabel('Pressure Enclosure 2 (Pa)')
+ax2.set_ylim(bottom=0)
+
+# Combine legends
+lines_left  = line1 + line2 + line3
+lines_right = line4 + line5 + line6
+all_lines = lines_left + lines_right
+all_labels = [l.get_label() for l in all_lines]
+
+ax.legend(all_lines, all_labels, loc='best')
+fig.savefig('ver-1kc_comparison_time_k10.png', bbox_inches='tight', dpi=300)
+
+# ax.plot(TMAP8_time_k10, TMAP8_pressure_H2_enclosure_1_k10, label=r"H$_2$ Enclosure 1", c='tab:red', linestyle='-')
+# ax.plot(TMAP8_time_k10, TMAP8_pressure_H2_enclosure_2_k10, label=r"H$_2$ Enclosure 2", c='tab:blue', linestyle='-')
+# ax.plot(TMAP8_time_k10, TMAP8_pressure_T2_enclosure_1_k10, label=r"T$_2$ Enclosure 1", c='tab:orange', linestyle='--')
+# ax.plot(TMAP8_time_k10, TMAP8_pressure_T2_enclosure_2_k10, label=r"T$_2$ Enclosure 2", c='tab:green', linestyle='--')
+# ax.plot(TMAP8_time_k10, TMAP8_pressure_HT_enclosure_1_k10, label=r"HT Enclosure 1", c='tab:purple', linestyle='-')
+# ax.plot(TMAP8_time_k10, TMAP8_pressure_HT_enclosure_2_k10, label=r"HT Enclosure 2", c='tab:cyan', linestyle='-')
+# ax.yaxis.set_major_formatter(plt.FuncFormatter(lambda val, pos: '{:.1e}'.format(val)))
+# ax.set_xlabel('Time (s)')
+# ax.set_ylabel('Pressure (Pa)')
+# ax.set_xlim(0, TMAP8_time_k10.max())
+# ax.set_ylim(bottom=0)
+# ax.legend(loc="best")
+# ax.grid(which='major', color='0.65', linestyle='--', alpha=0.3)
+# fig.savefig('ver-1kc_comparison_time_k10.png', bbox_inches='tight', dpi=300)
```

Comment:
Make sure only to add things that are needed. Lines that are commented out should generally be removed (or un-commented out if needed)